src/inference/AdvancedSMC.jl

```diff
@@ -267,7 +267,7 @@ end
 
 function assume(spl::Sampler{<:Union{PG,SMC}}, dist::Distribution, vn::VarName, ::VarInfo)
     vi = current_trace().vi
-    if isempty(getspace(spl.alg)) || vn.sym in getspace(spl.alg)
+    if vn in getspace(spl)
```

Comment:
What about the case where the space is `()`?

Reply:
There's no need to take of this anywhere (e.g., in the `assume` implementations) since the implementation of `in` in `RandomVariables.jl` checks
https://github.com/TuringLang/Turing.jl/blob/7e2e159184c2e1846fbb780c6b7bcd38b1944029/src/core/RandomVariables.jl#L611

Reply:
Cool, forgot that! Thanks.